bas/Model/Recorders.bas

```diff
@@ -447,244 +447,851 @@ source "../Scripts/Recorders.tcl"; # user recorders
 *#
 *# ShellMITC4
 *#
+
```

Comment:
Since this addition to this file is rather large, kindly add a commend with your name and title of your modification so it's easy to access in the future: 
*#begin Tejeswar 2021: title_of_your_modification
*#endTejeswar 2021: title_of_your_modification

Reply:
Nice idea! I will follow your suggestion. Thank you.